src/components/ec-summary-info/ec-summary-info.vue

```diff
@@ -0,0 +1,131 @@
+<template>
+  <div
+    class="ec-summary-info"
+    data-test="ec-summary-info"
+  >
+    <div
+      v-if="iconName"
+    >
+      <ec-icon
+        class="ec-summary-info__icon"
+        data-test="ec-summary-info__icon"
+        :class="{'tw-mr-12': iconName}"
+        :name="iconName"
+        :size="12"
+      />
+    </div>
+
+    <div class="ec-summary-info__content">
+      <div
+        v-if="title"
+        class="ec-summary-info__title"
+        data-test="ec-summary-info__title"
+      >
+        {{ title }}
+      </div>
+
+      <div
+        v-if="lineItems.length > 0 || hasSlot('default')"
+        data-test="ec-summary-info__content-lines"
+        class="ec-summary-info__content-lines"
+      >
+
+        <div
+          v-for="item in lineItems"
+          :key="item.text"
+          :class="[item.cssClasses]"
+          class="ec-summary-info__content-line-item"
+        >
+          <span
+            :class="[item.textCssClasses]"
+          >
+            {{ item.text }}
+          </span>
+
+          <span
+            v-if="item.iconName"
+            v-ec-tooltip="{
+              content: item.tooltipText
+            }"
+            :class="{'tw-ml-4': item.iconName}"
+          >
+            <ec-icon
+              class="ec-summary-info__content-line-item-icon"
+              :class="[item.iconCssClasses]"
+              :name="item.iconName"
```

Comment:
I'm not sure we should support this iconName here for now, maybe later if it will be needed, now I can't find any other icon except `i` icon.

Reply:
ok, changed it